ci/cases/gfsv17/obs_list.yaml

```diff
@@ -0,0 +1,17 @@
+observers:
```

Comment:
Why is this obslist in global-workflow? 
This is part of GDASApp.

Reply:
@guillaumevernieres is there an obs_list in gadsapp this can point to instead?

Reply:
We could, but we'd have to edit it before running. What's wrong with having the obs list here @aerorahul ?

Reply:
There is no need to edit anything.

Replace
```
SOCA_OBS_LIST: {{ 'PWD' | getenv }}/obs_list.yaml
```
with
```
SOCA_OBS_LIST: {{ HOMEgfs }}/sorc/gdas.cd/PATH_TO_ocean_obs_list.yaml
```
in the yaml

Reply:
I meant editing the obs_list.yaml @aerorahul , but that's not relevant to the proposed ci test, sorry.
That should do it @Jeeixner-NOAA :
```yaml
SOCA_OBS_LIST: {{ HOMEgfs }}/sorc/gdas.cd/parm/soca/obs/obs_list.yaml
```